src/ipc/mem/PageStack.h

```diff
@@ -116,10 +116,24 @@ class PageStack
     /// the number of (free and/or used) pages in a stack
     typedef unsigned int PageCount;
 
-    PageStack(const PoolId aPoolId, const PageCount aCapacity, const size_t aPageSize);
+    // XXX: poolId, pageSize look misplaced due to messy separation of PagePool
+    // (which should support multiple Segments but does not) and PageStack
+    // (which should not calculate the Segment size but does) duties.
+    /// PageStack construction and SharedMemorySize calculation parameters
+    class Config {
+    public:
+        uint32_t poolId = 0; ///< pool ID
+        size_t pageSize = 0; ///< page size, used to calculate shared memory size
+        PageCount capacity = 0; ///< the maximum number of pages
+
+        /// whether a newly created PageStack should contain capacity IDs
```

Comment:
What are "capacity IDs"? they do not seem to be defined.

Reply:
These are IDs below the requested capacity. There is some explanation in IdSet::makeFullBeforeSharing().

Reply:
Actually, there are no "capacity IDs".

There are many ways to parse createFull description. The correct way is "whether a newly created PageStack should contain n IDs (where n is equal to Config::capacity)". In branch commit eb6fd09, I adjusted the phrase in hope to decrease the number of ways it can be misparsed. Please feel free to adjust further, of course.